opal/mca/threads/pthreads/threads_pthreads_mutex.h

```diff
@@ -115,8 +115,6 @@ OPAL_DECLSPEC OBJ_CLASS_DECLARATION(opal_recursive_mutex_t);
     }
 #endif
 
-#endif
```

Comment:
why did you remove this endif?  it looks correct to me.  It closes the defined(OPAL_PTHREAD_RECURSIVE_MUTEX_INITIALIZER) no?

Reply:
This change should have been done to `qthreads/threads_qthreads_mutex.h` instead. Fixed.